.github/workflows/tests@v1.yml

```diff
@@ -139,10 +139,8 @@ jobs:
 
       - name: Setup environment
         run: |
-          sudo sh -c "echo 'deb http://security.ubuntu.com/ubuntu xenial-security main' >> /etc/apt/sources.list"
-          sudo apt-get update
-          sudo apt-get install libssl1.0.0
           pip3 install https://github.com/scylladb/scylla-ccm/archive/master.zip
+          sudo sh -c "echo 2097152 >> /proc/sys/fs/aio-max-nr"
```

Comment:
Why change that here?

Reply:
Just in case, cassandra also opens up lot's of files.